client/src/types/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
i think there's a version that it will disable the rule just for 1 line rather than the entire file
i prefer this so that the code is closer by and more easily associated to the unused var
that way we don't delete/use the var and forget to remove this